src/current/_includes/releases/v23.2/v23.2.7.md

```diff
@@ -0,0 +1,77 @@
+## v23.2.7
+
+Release Date: July 2, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-7-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Changefeeds]({% link v23.2/change-data-capture-overview.md %}) now defaults to using the bulk oracle for planning, which distributes work evenly across all replica in the locality filter, including followers if enabled. To disable this behavior and use the previous bin-packing oracle, set the [cluster setting]({% link v23.2/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. [#124925][#124925]
+- [`ALTER CHANGEFEED`]({% link v23.2/alter-changefeed.md %}) no longer removes the CDC query when [modifying changefeed properties]({% link v23.2/create-and-configure-changefeeds.md %}#modify-a-changefeed). [#125437][#125437]
+- The new metric `changefeed.usage.table_byte` allows you to observe billing for changefeed usage in CockroachDB {{ site.data.products.cloud }}. [#125525][#125525]
```

Comment:
```suggestion
```

I need to let the team know that we should remove any release notes relating to billing for now.